eng/pipelines/templates/stages/archetype-sdk-tests-isolated.yml

```diff
@@ -144,6 +150,8 @@ stages:
               - ${{ parameters.AdditionalSparseCheckoutPaths }}
             JobTemplatePath: /eng/pipelines/templates/jobs/live.tests.yml
             AdditionalParameters:
+              PreTestRunSteps:
```

Comment:
If you use PersistOidcToken option I think you remove these PreTestRunSteps completely.